src/niri.rs

```diff
@@ -5646,9 +5656,16 @@ impl Niri {
             }
 
             #[cfg(feature = "dbus")]
-            if let Err(err) = crate::utils::show_screenshot_notification(image_path) {
+            if let Err(err) = crate::utils::show_screenshot_notification(image_path.clone()) {
                 warn!("error showing screenshot notification: {err:?}");
             }
+
+            // Send screenshot completion event
+            let path_string = image_path
+                .as_ref()
+                .map(|p| p.to_string_lossy().into_owned());
```

Comment:
Can we send a PathBuf over the IPC? Does that even roundtrip through JSON? If not, then I think sending no path is better than a lossy path. Then the comment on the IPC event should also say that `None` means that the path is not available (either not saved to disk, or couldn't be encoded as a string).